tests/client.py

```diff
@@ -33,7 +33,8 @@ class TestTurbiniaClient(unittest.TestCase):
   @mock.patch('turbinia.client.task_manager.PSQTaskManager.setup')
   def testTurbiniaClientInit(self, _):
     """Basic test for client."""
-    config.LoadConfig() client = TurbiniaClient()
+    config.LoadConfig()
```

Comment:
That's.... Weird.  Don't know how that was submitted that way.  Might have happened when I was resolving merge commits somehow.